main.go

```diff
@@ -84,8 +107,31 @@ func main() {
 		http.HandleFunc(*metrics, metricHandler(services))
 	}
 
+	tcp := ip4or6(*bind)
+
+	l, err := net.Listen(tcp, *bind)
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	log.Infoln("Starting HTTP server on", *bind)
-	log.Fatal(http.ListenAndServe(*bind, nil))
+	log.Fatal(http.Serve(l, sm))
+}
+
+func ip4or6(s string) string {
```

Comment:
Would be possible to move this function into a utils.go file or module alongside with removeElements?

Reply:
done.
Please fix comment for exporters.RemoveElements if it is not clear description.